### ISSUE TITLE
[Weasyl] Tags are not showing up in metadata file

### ISSUE THREAD

**Issue by Eupeptic1** [Simon-Tesla/RaccoonyWebEx#49]
## Quick version

When downloading from Weasyl, the tags don't show up in the metadata file anymore.

## Long version / steps to reproduce:

1.  Open a submission page on Weasyl in your browser.  Make a note of the artist's name and the tags shown on Weasyl.

2.  Mouse over the raccoon-head icon, click the gear, turn "download metadata file" on, and save changes.

3.  Mouse over the raccoon-head icon, and click "Download".

4.  In your OS file manager, navigate to where Raccoony saves files (probably `~/Downloads/raccoony/weasyl` ), and then to the subdirectory with the name of the artist.

5.  Look at the metadata file corresponding to the image you just downloaded.

Expected result: "tags" element in the JSON contains the same tags as observed on the site.

Actual result: "tags" element contains an empty array, [].

## Versions

This is with Raccoony 1.2.2 and Firefox Quantum 68.4.1esr, 64-bit, on Linux desktop.

**Comment by Eupeptic1** [comment]
## Analysis

The Weasyl plugin currently does this to get the tags

`let tags = querySelectorAll("#di-tags .tags a")
      .map(el => el.textContent.trim());`

On Weasyl, there is still a `<div class="di-tags">`, that has the tag title and the individual tags.  There is another div inside that div, named `<div class="tags open clear">`, that actually contains the individual tags.  (Yes, there are spaces in the div name.)  I suspect maybe this div was previously named just `class="tags"`, which would have worked with the existing plugin code.  I tried a few variations on the syntax to get it to pick up the new div, but I don't know enough querySelector() to get it right.

**Comment by Simon-Tesla** [comment]
Oh, given that DOM snippet, it looks like they changed the ID to a class. The selector `".di-tags .tags a"` would probably work.

**Comment by Eupeptic1** [comment]
> The selector `".di-tags .tags a"` would probably work.

I tried that as the selector, and it worked OK on three images from the front page so far.  Both the "tags" element in the JSON and the "Tags:" line in the human-readable form are populated correctly now.  (I didn't notice that the tags were also blank in the human-readable section when I first reported the bug.)

Line 107 of weasyl.ts, plus or minus.
